src/components/form-elements/Input.jsx

```diff
@@ -3,9 +3,9 @@
 import React from 'react';
 import classnames from 'classnames';
 
-type InputSizeType = 'small' | 'normal' | 'large';
+type InputSizeType = 'normal' | 'large';
 
-type InputColorType = 'normal' | 'light' | 'light-alt';
+type InputColorType = 'default' | 'white';
```

Comment:
so in one place we have `normal` and in other `default`, why is that?

Reply:
nope, colors are set to `default` and `white` and sizes to `normal` and `large`

Reply:
is that a updated standard for colors? there was `normal` previously in component

Reply:
yeah, this would be the new approach for form elements.